src/init.js

```diff
@@ -73,6 +78,13 @@ export const init = async (options) => {
 				choices: ['preact'],
 				default: 'preact',
 			},
+			{
+				type: 'list',
+				name: 'template',
+				message: "Please choose the template you'd like to use:",
+				choices: repos.data.filter((repo) => repo.name.startsWith('snapfu-template-')),
```

Comment:
I think these choices should be limited based on the answer to the previous question. If `preact` is the chosen framework, then only `snapfu-template-preact-` templates should be shown as options.

Reply:
The default would also have to be dynamic based on the answer to the previous question.